### DISCUSSION TITLE
Could not install

### DISCUSSION THREAD

**Discussion by Crabmeat1432** [Flow-Launcher/Flow.Launcher#2124] [General]
The lastest install program from 2 days ago would not even install on my brand new Windows 11 computer.  Zippo. Nada....

**Comment by Garulf** [comment]
Hello @Crabmeat1432 sorry to hear you’re having problems installing Flow Launcher. 

Do you have a screenshot of the error you’re having?

**Reply by Crabmeat1432** [reply]
Hi, I found out afterward that it actually did install.  It must have been done in the background or something.  Looks interesting.  I'm wondering if I'll ever have the motivation to fully explore all of its potential.  It's pretty in-depth.  I'll have to set some time aside.
